docs/CONFIGURATION.md

```diff
@@ -87,9 +87,9 @@ With `excluded-mutations`, you can turn off certain mutations in the project. Al
 **Description:**
 Specify the thresholds for mutation scores.
 
-* `mutation score >= high`: Success! Mutation score will be logged at the INFO level.
-* `high > mutation score >= low`: Warning! Mutation score will be logged at the WARN level.
-* `mutation score < low`: Dangerously low! Mutation score will be logged at the ERROR level with an extra warning.
-* `mutation score < break`: Error! Stryker will exit with exit code 1, indicating a build failure.
+- `mutation score >= high`: Success! Mutation score will be logged at the INFO level.
+- `high > mutation score >= low`: Warning! Mutation score will be logged at the WARN level.
+- `mutation score < low`: Dangerously low! Mutation score will be logged at the ERROR level with an extra warning.
+- `mutation score < break`: Error! Stryker will exit with exit code 1, indicating a build failure.
```

Comment:
Maybe we should make it a textual description instead of the `>=` etc what do you think?